src/test/resources/org/opa/ideaplugin/lang/parser/fixtures/imports.rego

```diff
@@ -1,6 +1,6 @@
 package test
 
-import someOther.policy
+import data.policy
```

Comment:
why does it matter?

Reply:
tldr: every import have to start from import or data

https://www.openpolicyagent.org/docs/latest/policy-language/#imports

Reply:
I don't think that's what they meant. The docs say `data` and `input` are implicitly imported. @anderseknert can you weigh in?

Reply:
All data (with the sole exception of the `input` document) is ultimately merged into the `data` document (under `/v1/data`) so importing anything from outside of that doesn't really make sense - @SuchSkill is right on that. Now whether the _parser_ should take this into account though? I don't know :)